moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
Looks good to me. Eigen is completely header only anyway. But out of interest, what warning did this generate?